dev/io.openliberty.security.oidcclientcore.internal/resources/io/openliberty/security/oidcclientcore/internal/resources/OidcClientCoreMessages.nlsprops

```diff
@@ -48,3 +48,32 @@ ERROR_BUILDING_AUTHORIZATION_ENDPOINT_URL.useraction=See the error in the messag
 AUTHENTICATION_RESPONSE_ERROR=CWWKS2407E: The {0} OpenID Connect client encountered the following error while verifying the authentication response from the OpenID Connect provider: {1}
 AUTHENTICATION_RESPONSE_ERROR.explanation=The response from the OpenID Connect provider might be missing information. There might be a problem with the code returned from the OpenID Connect provider.
 AUTHENTICATION_RESPONSE_ERROR.useraction=See the error in the message for more information.
+
+CALLBACK_MISSING_STATE_PARAMETER=CWWKS2408E: The callback request from the OpenID Connect provider is missing the state parameter.
+CALLBACK_MISSING_STATE_PARAMETER.explanation=The authorization request to the OpenID Connect provider might not have included a state parameter.
+CALLBACK_MISSING_STATE_PARAMETER.useraction=Ensure that the authorization request to the OpenID Connect provider includes a state parameter.
+
+STATE_VALUE_IN_CALLBACK_INCORRECT_LENGTH=CWWKS2409E: The [{0}] state parameter that is in the callback request from the OpenID Connect provider is not the correct length. The parameter is expected to be {1} characters long.
+STATE_VALUE_IN_CALLBACK_INCORRECT_LENGTH.explanation=
```

Comment:
are these intentionally left blank?

Reply:
Not intentional, no. I was writing the messages and the code together and was going to go back to finish the remaining messages all at the same time. Forgot to do that before I opened the PR. I've now included the missing entries.